osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -28,6 +28,16 @@ public class SkinnableSound : SkinReloadableDrawable
         public override bool RemoveWhenNotAlive => false;
         public override bool RemoveCompletedTransforms => false;
 
+        /// <summary>
+        /// Whether to play the underlying sample when aggregate volume is zero.
+        /// Note that this is checked at the point of calling <see cref="Play"/>; changing the volume post-play will not begin playback.
+        /// Defaults to false unless <see cref="Looping"/>.
+        /// </summary>
+        /// <remarks>
+        /// Can serve as an optimisation if it is known ahead-of-time that this behaviour will not negatively affect behaviour.
```

Comment:
this reads a little loopy (*behaviour will not negatively affect behaviour*). How about something like

```suggestion
        /// Can serve as an optimisation if it is known ahead-of-time that this behaviour is allowed in a given use case.
```

(also not 100% sure what sort of optimisation would this be but it might just be that it's late and my brain is gone).

Reply:
It's a bit of an odd one.

- Doesn't play samples at a bass level when fast forwarding (helps with performance)
- Doesn't play zero volume samples (causing "clicks" for some audio hardware)